cve_bin_tool/util.py

```diff
@@ -79,13 +79,13 @@ def regex_find(lines, version_patterns) -> str:
     """Search a set of lines to find a match for the given regex"""
     new_guess = ""
 
-    for line in lines:
-        for pattern in version_patterns:
-            match = pattern.search(line)
-            if match:
-                new_guess2 = match.group(1).strip()
-                if len(new_guess2) > len(new_guess):
-                    new_guess = new_guess2
+    for pattern in version_patterns:
+        match = pattern.search(lines)
+        if match:
+            new_guess2 = match.group(1).strip()
+            if len(new_guess2) > len(new_guess):
+                new_guess = new_guess2
+                break
```

Comment:
```suggestion
            new_guess = match.group(1).strip()
            break
```
This `new_guess2` is kinda useless if you `break` now, every match will be longer than an empty string so you can simplify this.